tests/integration/test_models/test_full_battery_models/test_lithium_ion/base_lithium_ion_tests.py

```diff
@@ -20,7 +20,10 @@ def test_basic_processing(self):
     def test_sensitivities(self):
         model = self.model()
         param = pybamm.ParameterValues("Ecker2015")
-        modeltest = tests.StandardModelTest(model, parameter_values=param)
+        solver = pybamm.IDAKLUSolver()
```

Comment:
it's a bit ugly but this could be a `if pybamm.have_idaklu()` to use the IDAKLU solver for ubuntu tests and casadi solver for windows/mac tests

Reply:
agreed, this works